plugins/reporters/trustsource/src/main/kotlin/TrustSourceModel.kt

```diff
@@ -22,34 +22,53 @@ package org.ossreviewtoolkit.plugins.reporters.trustsource
 import kotlinx.serialization.Serializable
 
 @Serializable
-data class TrustSourceModule(
+data class NewScan(
+    /** Name of the module */
     val module: String,
-    val moduleId: String,
-
-    val dependencies: List<TrustSourceDependency>
+    /** SCM tag of the module */
+    val tag: String = "",
+    /** SCM branch of the module */
+    val branch: String = "",
```

Comment:
Would it be valid if both `tag` and `branch` are empty? If not, the docs should probably say that either or is required. Also, what if only a Git commit SHA1 is known, where would that go?

Reply:
Added comments and the commit field, thanks for the advise. These fields are normally filled by a developer submitting a scan of a module, hence nobody asked before to provide this info.